docs/content/usage/renovate.md

```diff
@@ -0,0 +1,44 @@
++++
+title = "Renovate"
+weight = 110
++++
+
+[Renovate](https://docs.renovatebot.com/) is an open source dependency update tools. It now supports the followings:
+
+* Update Hermit Packages
+* Using Hermit as a source of Binaries
+
+## Update Hermit Packages
+
+Package update with Renovate ensures updates are done explicitly to the code repository via code commits. Together with proper default branch protection setup & CI pipeline steps, it can prevent problematic hermit package updates flows into the repositoryto break the build, which is always a problem in the implicit package update using [Channel](../updates).
```

Comment:
```suggestion
Package update with Renovate ensures updates are done explicitly to the code repository via code commits. Together with proper default branch protection setup & CI pipeline steps, it can prevent breaking hermit package update flows into the repository, which is always a problem in the implicit package update using [Channel](../updates).
```